### DISCUSSION TITLE
Front End Submit deactivates

### DISCUSSION THREAD

**Discussion by dietrichmd** [liedekef/events-made-easy#254] [Q&A] [answered]
Not sure if I missed something along the way, but with the newer versions of EME, I cannot install the Front End Submit plugin.  Every time it spews out the error

`The Events Made Easy Frontend Submit plugin is an extension to the Events Made Easy plugin, which has to be installed and activated first. The plugin has been deactivated.`

Yet, EME is, in fact, installed, activated, and working fine.

Any ideas?

**Comment by liedekef** [comment]
Is the plugin Events Made Easy installed in the plugin path called "events-made-easy" ?
The frontend plugin checks for EME using this logic:

`if (is_plugin_active('events-made-easy/events-manager.php'))`

Or - can it be EME is network installed? Because indeed I didn't take that into account ... this change (for EME frontend submit) should solve that: https://plugins.trac.wordpress.org/changeset/2945462/

**Comment by dietrichmd** [comment]
Problem was the installed directory.  The ZIP from github puts everything into events-made-easy-main.

**Reply by liedekef** [reply, marked as answer]
If downloadin from github, you need to download a released version. Clicking on "Releases" gives you: https://github.com/liedekef/events-made-easy/releases
Downloading from there will give you a zip that unpacks in just "events-made-easy" ...

**Reply by dietrichmd** [reply]
got it.  I just renamed the folder and all is well.  May want to put a lil note in the readme for the frontend plugin about that -- just to avoid the 'me's' of the world :)
